src/representation/molecularsurface-representation.ts

```diff
@@ -274,9 +274,13 @@ class MolecularSurfaceRepresentation extends StructureRepresentation {
         })
       )
 
-      const doubleSidedBuffer = new DoubleSidedBuffer(surfaceBuffer)
-
-      bufferList.push(doubleSidedBuffer)
+      if (this.getBufferParams().side == 'double') {
+        const doubleSidedBuffer = new DoubleSidedBuffer(surfaceBuffer)
+        bufferList.push(doubleSidedBuffer)
+      }
+      else {
+        bufferList.push(surfaceBuffer)
+      }
```

Comment:
@garyo - this looks great, but one thing I don't quite understand here. Stick a couple of breakpoints at line 278 and 282 to see what I mean:

If I load the representation/molecularsurface-av example it draws two surfaces in contour mode (so this section of code isn't reached at all - so far so good). 

Then if I set the side to 'front' and turn off contour (in that order), it just pushes "surfaceBuffer" onto bufferList (no DoublesidedBuffer - still good).

Then if I switch the side to 'double' - it just works! but DoubleSidedBuffer was never called... So, is DoubleSidedBuffer actually not required?

NB: If I load in contour mode, leave it on double and turn off contours, a DoubleSidedBuffer is created, but I can't tell the difference visually between the two results..